fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -111,6 +113,8 @@
 
     private static final Logger LOGGER = getLogger(FedoraResourceImpl.class);
 
+    private static final PropertyConverter propertyConverter = new PropertyConverter();
```

Comment:
Since you are the one doing the suggesting, and you don't know, then probably not.